### PR TITLE
Added P1 Orientation 2024

### PR DESCRIPTION
**Pull request by gil-tan**
Added P1 Orientation 2024

### CODE REVIEW

**Comment by JeremyEPPS** [comment]
Looks good to go.

File: files/P1_Orientation_2024.pdf
Comment (JeremyEPPS) [inline files/P1_Orientation_2024.pdf:1]: looks good to go

**Review by JeremyEPPS — COMMENTED** [review]
Looks good to go

**Review by EPAdm00 — APPROVED** [review]
Good to Go

**Pull request merged**